### ISSUE TITLE
Plugin exclude does not work

### ISSUE THREAD

**Issue by jdelvecchio** [lean-delivery/ansible-role-sonarqube#5447]
##### SUMMARY

Cannot exclude plugins from default plugin list.

##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME

Plugin exclusion with var sonar_default_excluded_plugins or sonar_excluded_plugins

##### ANSIBLE VERSION

```
2.15.12
```

##### CONFIGURATION

N/A

##### OS / ENVIRONMENT

Ubuntu 2404 on host and bastion

##### STEPS TO REPRODUCE

Use your latest exemple playbook on a Ubuntu2404 with ansible 2.15.12 -> error on csharp plugin, and cannot exclude it even if I specify it in the corresponding variables

```yaml
your latest example playbook with modified excluded vars
```

##### EXPECTED RESULTS

Plugin is correctly excluded

##### ACTUAL RESULTS

Plugin is not excluded, sonar fails to launch

```
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'jdk.internal.loader.ClassLoaders$AppClassLoader@5ffd2b27-org.sonar.server.plugins.ServerPluginManager': Initialization of bean failed; nested exception is Found a plugin 'C# Code Quality and Security' in the directory 'extensions/plugins' with the same key [csharp] as a built-in feature 'C# Code Quality and Security'. Please remove 'extensions/plugins/sonar-csharp-plugin-9.10.0.77988.jar'.
```

**Comment by dzmitry-rudnouski** [comment]
@jdelvecchio, share the full playbook please.

**Comment by jdelvecchio** [comment]
playbook :
```yaml
---
- name: Install SonarQube
  hosts: devops_prod_sonar
  become: true
  roles:
    - role: lean_delivery.java
    - role: geerlingguy.postgresql
    - role: lean_delivery.sonarqube
```


vars file : 
```yaml
---
sonar_db_password: superpass

sonar_db:
  type: postgresql
  port: 5432
  options: ''
  host: localhost
  name: sonar
  user: sonar
  password: "{{ sonar_db_password }}"

sonar_web_password: magicpass

sonar_version: 10.7.0.96327
sonar_check_url: 'http://{{ ansible_fqdn }}:9000'
sonar_proxy_server_name: sonarqube.org.com
sonar_install_recommended_plugins: false
sonar_install_optional_plugins: false

sonar_default_excluded_plugins: 
  - extensions/plugins/sonar-csharp-plugin-9.10.0.77988.jar

# or i've also tried this and it does not seem to exclude it either.
# sonar_excluded_plugins: 
# - extensions/plugins/sonar-csharp-plugin-9.10.0.77988.jar


change_password: true
sonar_migrate_db: false  # set to true if updating SonarQube to new version
```

So with the above, I get the error described, the plugin is not excluded. I have to set : 
```yaml
sonar_default_plugins: []
```
for the play to work (or just list the ones that works but I didn't bother since I don't really need them)

**Comment by dzmitry-rudnouski** [comment]
Fixed in 2.16.0.
Usage example:
```
sonar_default_excluded_plugins: 
  - sonar-csharp-plugin-<version>.jar
```
